admin/class-monk-admin.php

```diff
@@ -207,15 +207,15 @@ public function monk_general_settings_render() {
 	 * @return  void
 	 */
 	public function monk_settings_tabs() {
-		$url     = home_url() . $_SERVER['REQUEST_URI'];
+		$url     = is_ssl() ? 'https://' : 'http://';
```

Comment:
Use `admin_url` function to generate admin-related links.
https://codex.wordpress.org/Function_Reference/admin_url